src/components/Support/form/NewRequestForm.tsx

```diff
@@ -38,18 +38,24 @@ function NewRequestForm() {
   });
   const [isHealthy, setIsHealthy] = useState<boolean>(false);
   const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(true);
 
-  const handleInputChange = (name: string, value: string | FileList) => {
+  const handleInputChange = (name: string, value: string) => {
     setFormValues((prevValues) => ({
       ...prevValues,
       [name]: value,
     }));
+
+    if (name === 'comment') {
+      setIsButtonDisabled(value.trim().length < 30);
```

Comment:
Only update state when required.  Not on every user char input.

```suggestion
      const shouldBeDisabled = value.trim().length < 30;

      if (shouldBeDisabled !== isButtonDisabled) {
        setIsButtonDisabled(shouldBeDisabled);
      }
```